tests/e2e_tests/.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends": ["eslint:recommended", "plugin:@typescript-eslint/eslint-recommended", "plugin:@typescript-eslint/recommended"],
+  "rules": {
+    "prettier/prettier": 2,
+    "consistent-this": ["error", "self"],
+    "import/no-named-as-default": 0,
+    "no-console": 0,
+    "no-multiple-empty-lines": 2,
```

Comment:
can we be explicit here? suggested: `"max": 2` also: could we have `"maxEOF":1` and `"maxBOF":0` as I believe this are the max we use.

Reply:
that will be enforced from prettier already 👍